pkg/tfvars/openstack/bootstrap_ignition.go

```diff
@@ -89,6 +89,20 @@ func generateIgnitionShim(userCA string, clusterID string, swiftObject string) (
 		},
 	}
 
+	// Openstack Ca Cert file
+	openstackCAFile := ignition.File{
+		Node: ignition.Node{
+			Filesystem: "root",
+			Path:       "/opt/openshift/tls/cloud-ca-cert.pem",
+		},
+		FileEmbedded1: ignition.FileEmbedded1{
+			Mode: &fileMode,
+			Contents: ignition.FileContents{
+				Source: fmt.Sprintf("data:text/plain;base64,%s", b64.StdEncoding.EncodeToString([]byte(userCA))),
```

Comment:
(On mobile, so it’s difficult to check) are we manually mangling this together elsewhere? I’m surprised we aren’t using a dataurl library for this.

Reply:
haha I didnt know that exists, yeah we have been manually formatting our dataurls.

Reply:
As ugly as this code might be, I would rather address that in a seperate pull request/feature

Reply:
Can you please open a bug or issue to follow up on that? Whichever will help you track it most effectively is fine.

Reply:
I've created https://bugzilla.redhat.com/show_bug.cgi?id=1794640 for tracking it.

Reply:
the installer already vendors `github.com/vincent-petithory/dataurl`... Updating the code to use the library should be trivial to do in this PR.